.github/workflows/pr_scan.yml

```diff
@@ -36,15 +36,33 @@ jobs:
       run: |
         Scripts/validate-Adyen-SDK-version.sh
 
-    - name: Build and test
+    - name: Clean Build and Unit Test
       run: |
         xcodebuild -version
-        xcodebuild clean build test -project "${project}" -scheme "${scheme}" ${params} -destination "${destination}" -skipPackagePluginValidation | xcpretty --utf --color && exit ${PIPESTATUS[0]}
+        xcodebuild clean build test -project "${project}" -scheme "AdyenUIKitTests" ${params} -destination "${destination}" -skipPackagePluginValidation -parallel-testing-enabled NO | xcpretty --utf --color && exit ${PIPESTATUS[0]}
       env:
         project: 'Adyen.xcodeproj'
         params: '-derivedDataPath ./DerivedData -enableCodeCoverage YES'
-        scheme: 'AdyenUIHost'
-        destination: 'name=iPhone 14'
+        scheme: ''
```

Comment:
You can just set `"AdyenUIKitTests"`  here

Reply:
Ahh yeah good one, came from copy-pasting during troubleshooting :D